### ISSUE TITLE
install doc of c_api is out of date

### ISSUE THREAD

**Issue by Greastate** [facebookresearch/faiss#2421]
Faiss version:  dbc3d1d54bb17a7c4a67c6b6ba014dc10d2314ef

The install documentation of c_api is outdated, and the compilation cannot be successfully executed according to the steps inside. Now the compilation of c_api can be completed by adding compilation options(-DFAISS_ENABLE_C_API=ON), instead of entering the c_api directory and executing the make command. Please update the documentation, or delete c_api/install.md. , the confusion caused by the current documentation will waste a lot of time for users.

Thinks!

**Comment by Enet4** [comment]
True, the details in c_api/INSTALL.md were applicable before the project started using CMake, and it has not been updated since then. Would you like to send in a pull request with the changes that you propose?
